### PR TITLE
Upstream Merge July 1rst

### PR DESCRIPTION
**Pull request by Jolly-66**
ok i hope to fucking god this isn't dumb and stupid
this won't alter the upstream pull schedule btw

|image
image
https://www.youtube.com/watch?v=czTksCF6X8Y&t=4s&ab_channel=NickCagnetti

### CODE REVIEW

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -57,7 +57,7 @@
 #define BODYPART_ORGANIC   1
 #define BODYPART_ROBOTIC   2
 
-#define DEFAULT_BODYPART_ICON_ORGANIC 'jollystation_modules/icons/mob/human_parts_greyscale_js.dmi' //Changing from the "icons/mob/human_parts_greyscale" file to the "jollystation_modules/icons/mob/human_parts_greyscale_js" version, to account for Skrell additions. JollyStation Edit -Jon
+#define DEFAULT_BODYPART_ICON_ORGANIC 'icons/mob/human_parts_greyscale.dmi'
 #define DEFAULT_BODYPART_ICON_ROBOTIC 'icons/mob/augmentation/augments.dmi'
```
Comment (MrMelbert) [inline code/__DEFINES/mobs.dm:62]: another missed comment reee

File: code/__HELPERS/global_lists.dm [hunk 2]
```diff
@@ -34,7 +34,6 @@
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/moth_antennae, GLOB.moth_antennae_list)
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/moth_markings, GLOB.moth_markings_list)
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/tails/monkey, GLOB.tails_list_monkey)
-	init_sprite_accessory_subtypes(/datum/sprite_accessory/skrell_headtentacles, GLOB.skrellheadtentacles_list) //Adding Skrell Headtentacles init -Jon
```
Comment (MrMelbert) [inline code/__HELPERS/global_lists.dm:37]: again

File: code/modules/client/preferences.dm [hunk 3]
```diff
@@ -1985,12 +1985,6 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					to_chat(user, span_notice("OOC Commendation Heart disabled"))
 					save_preferences()
 
-				if("hear_speech_sounds")
-					hear_speech_sounds = !hear_speech_sounds
-
-				if("hear_radio_sounds")
-					hear_radio_sounds = !hear_radio_sounds
-
```
Comment (MrMelbert) [inline code/modules/client/preferences.dm:1993]: lost

**Review by MrMelbert — APPROVED** [review]
If it compiles and nothing looks weird or no on immediate runtimes, looks good to me

**Comment by Jolly-66** [comment]
hrnnn keybinds are still fucked

File: code/modules/client/preferences_savefile.dm [hunk 4]
```diff
@@ -208,10 +209,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	READ_FILE(S["pda_style"], pda_style)
 	READ_FILE(S["pda_color"], pda_color)
 
-	// Custom hotkeys
-	READ_FILE(S["key_bindings"], key_bindings)
-	check_keybindings()
-	// hearted
```
Comment (MrMelbert) [inline code/modules/client/preferences_savefile.dm:214]: I see

Comment (MrMelbert) [reply code/modules/client/preferences_savefile.dm:214]: Undelete and move this line of code to below the `hear_speech_sounds` and` hear_radio_sounds` client prefs

**Review by MrMelbert — APPROVED** [review]

**Pull request merged**